content/1.2-rc/docs/installation/requirements.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Requirements"
+date: 2024-11-25
+---
+
+The orchestrator runtime/deployment is made of two main parts: `sonataflow operator` and `RHDH operator`  
+SonataFlow operator resource requirements are described [here](https://sonataflow.org/serverlessworkflow/latest/cloud/operator/install-serverless-operator.html#_prepare_a_minikube_instance)  
+The operator deploys a Data Index service and a Jobs service.
+These are the recommended minimum resource requirements for their pods:
```

Comment:
I see value in having a quick and easy way to get the min resource requirements, do you think they may get outdated easily?

Reply:
I took it from our installation. there's no official reocmendation